app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -83,31 +83,21 @@ def start
   def pod_wait
     _log.info("waiting for pod #{pod_full_name} to be available")
 
-    client       = kubernetes_client
-    health_url   = pod_proxy_url(client, INSPECTOR_HEALTH_PATH)
-    http_options = {
-      :use_ssl     => health_url.scheme == 'https',
-      :verify_mode => ext_management_system.verify_ssl_mode,
-      :cert_store  => ext_management_system.ssl_cert_store,
-    }
-
-    # TODO: move this to a more appropriate place (lib)
-    response = pod_health_poll(client, health_url, http_options)
-
-    case response
-    when Net::HTTPOK
+    begin
+      ready = kubernetes_client.get_pod(options[:pod_name], options[:pod_namespace])[:status][:containerStatuses][0][:ready]
```

Comment:
Its by containers. Each container has one status, so if a pod had two containers in it this list would be of length 2. because we only have one container we can safely use `[0]`.